### ISSUE TITLE
NTR 'raw materials'

### ISSUE THREAD

**Issue by pdez90** [SDG-InterfaceOntology/sdgio#91]
Hi I looked for a definition of this online, but couldnt find any definition from a UN site. However I found this: https://stats.oecd.org/glossary/detail.asp?ID=1740
Natural resources are natural assets (raw materials) occurring in nature that can be used for economic production or consumption.

This is not an SDG term perse but its related to the Goal to the Sustainable Consumption and Production Goal

**Comment by pbuttigieg** [comment]
@pdez90 Thanks for retrieving the definition.
What's the difference between an asset and a resource that should be represented in the SDGIO?

**Comment by pdez90** [comment]
Hi,

I got this definition of 'asset' from the UN Stats Division: An asset is a
store of value representing a benefit or series of benefits accruing to the
economic owner by holding or using the entity over a period of time. It is
a means of transferring value from one accounting period to another.
http://unstats.un.org/unsd/envaccounting/londongroup/meeting16/LG16_8a.pdf

I cant find a definition for resource. Il ask the SDG focal point tomorrow

On Thu, Feb 18, 2016 at 10:35 PM, Pier Luigi Buttigieg <
notifications@github.com> wrote:

> @pdez90 <https://github.com/pdez90> Thanks for retrieving the definition.
> What's the difference between an asset and a resource that should be
> represented in the SDGIO?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/SDG-InterfaceOntology/sdgio/issues/91#issuecomment-185883008>
> .
>



-- 
Yours Sincerely,
Priyanka deSouza

**Comment by pbuttigieg** [comment]
Thanks! This distinction will be very helpful. It seems to me that an asset is some resource which confers "benefit" to its owner. The idea that it represents a benefit is a bit odd to me. I think it can be an agent in a process which provides its owner with more 'benefits'. Not sure what the distinction between 'owner' and 'economic owner' is yet, though.

**Comment by mark-jensen** [comment]
Our current definition for `asset`
An independent continuant under the control of some person or organisation which confers an economic benefit in some economic system.

**Comment by mark-jensen** [comment]
This links to discussion in #82

**Comment by pbuttigieg** [comment]
>Natural resources are natural assets (raw materials) occurring in nature that can be used for economic production or consumption. https://github.com/SDG-InterfaceOntology/sdgio/issues/91#issue-134594561

So we need "natural asset" with a synonym or preferred label as raw material for now. More discussion in https://github.com/SDG-InterfaceOntology/sdgio/issues/82#issuecomment-184310872 and https://github.com/SDG-InterfaceOntology/sdgio/issues/94#issuecomment-185815834; the latter contains:

>`natural asset` =def. An `asset` which is the result of or itself an environmental process that has not been substantially altered by human activity.

**Comment by mark-jensen** [comment]
`natural asset` with above def was added in https://github.com/SDG-InterfaceOntology/sdgio/commit/95f128d5b7afdeb5fc20826cccf4ec20f2e31670

I'll add the synonym annotation for now. As well as fix my previous uses of 'alternative term'. I had forgotten we previously agreed to 'synonym'.

@pdez90 Would you confirm if 'raw material' is the UNEP preferred label for `natural asset`, or if use of synonym is ok, or if there is some clear difference between the two?

**Comment by mark-jensen** [comment]
We are running into a problem here though, which I will take up in more detail in #82. `asset` is currently an `independent continuant`, but if we make `natural asset` a subclass, then an `occurrent` can also be an asset.

Also, the relationship between `resource` and `asset` now seems to be one of specialization, where the latter is a resource for which the benefit is economic or financial, i.e., it having value in an economic system.

**Comment by pbuttigieg** [comment]
>We are running into a problem here though, which I will take up in more detail in #82. asset is currently an independent continuant, but if we make natural asset a subclass, then an occurrent can also be an asset.

True, based on the definitions coming in, it seems assets can be occurrents too. We can bump this up a level or create "material asset" and "processual asset". These would all be filled by inference as a [material entity, process] bearing an asset role. Also see https://github.com/SDG-InterfaceOntology/sdgio/issues/94#issuecomment-187746261

>Also, the relationship between resource and asset now seems to be one of specialization, where the latter is a resource for which the benefit is economic or financial, i.e., it having value in an economic system. 

I agree with this, but see https://github.com/SDG-InterfaceOntology/sdgio/issues/94#issuecomment-187754862

**Comment by mark-jensen** [comment]
> As this inferred class can have such broad membership, we can bump this up a level (to a subclass of entity; extreme, I know, but that's what it looks like) or create "material asset", "processual asset", etc. These would all be filled by inference as a [material entity, process] bearing an asset role

A process can't bear a role.

It's the participants in processes that bear roles. The process realizes the role. I don't believe an education process, i.e., the process of being educated, is a resource, nor is some health care process. Is my cataract surgery a resource? But, rather, I think it's the systems that support such processes (indeed the material entities which compose such a system) which are the resource. My cataract surgery realizes the resource role which is borne by the health care system. The surgery process necessarily depends upon the system in some way.

Does that sound right? I did a quick draft of a diagram that can be expanded and refined to illustrate how we are putting this together. It is useful for #94 & #82. 

I am not yet sure how `ecosystem/environmental service`, or any service for that matter, fits.
How does `healthcare service` differ from `healthcare process` or `healthcare system`? I am leaning towards saying a service is the same as the system. It's the system that one has access to. We could expand the diagram to include `access` and and `basic service`, the inferred `use of a resource`, etc. Also need to figure out what kind of processes realize an `asset role`

![resource 2](https://cloud.githubusercontent.com/assets/11448917/13386729/987a16f2-de7d-11e5-9d5d-377c4da3c19e.png)

**Comment by pbuttigieg** [comment]
>A process can't bear a role...My cataract surgery realizes the resource role which is borne by the health care system. The surgery process necessarily depends upon the system in some way.

Quite right. This works and supports the intention behind many goals: you can't preserve a process which has value without preserving its underlying support systems: i.e. you can't sustain timber harvesting without sustaining the source forest(s). 

But the issue remains: the wording we face in the literature deals with processes and material entities simultaneously. We have to create a union class which contains material assets (material entities that bear an asset role); processes which generate or increase the value of assets, or reduce the costs of some economic process; and more deontic entities whose realisation is of value in an economic system (e.g. rights). 

This suggests that "processual assets" need their own defined, union class, along the lines of:

`processual asset` = A process which produces assets, increases the value of existing assets, leads to the realisation of non-material assets, or reduces the costs associated with some economic process.
* `process` and `has output` some `asset`
* or, more generally: `process` and `has output` some `resource`
* Also: `process` and `has input` some `undesirable thing` and 'has output' some `not so undesirable thing`

However, some valuable outputs are not assets (more like "benefits") and thus are not necessarily of direct economic value. We should be mindful of this and not restrict `ecosystem service` and related classes to the production of economic assets. There are cultural benefits and so on.

>How does healthcare service differ from healthcare process or healthcare system? I am leaning towards saying a service is the same as the system.

I wouldn't say so. The system is a material entity (a forest, a water treatment system, a healthcare system), not a processual one. Systems or system parts can participate in processes (forest-mediated carbon fixation, waste processing, distribution of vaccines). Some of these processes are beneficial to humans and are thus services. If we avoid roles, we can create defined or union classes as above.

A service is a specialisation of a process: some process which "aids" or "benefits" humans (typically).
Let's consider a couple of ecosystem services. 
* production of clean drinking water: This would be a process with an output that is a resource. 
* decomposition of wastes. Some of these processes have hazardous materials as inputs and non-hazardous materials as outputs. They could also output resources.
In a way, this could be a superclass of "processual asset" which would be a specialise "benefit" to "asset"

**Comment by cfrancois7** [comment]
Here which gathered many definition of what is resources from different field of research or statistic administration.
[The paper is free](http://onlinelibrary.wiley.com/doi/10.1111/jiec.12386/suppinfo)
